src/main/webapp/script.js

```diff
@@ -289,4 +302,104 @@ function toggleSidebar() {
         toggleButton.innerHTML = "<i class=material-icons>navigate_before</i>";
         toggleButton.style.left = "300px";
     }
+}
+
+// This function returns the aqi from the data at the specified index.
+// If the aqi is unknown at that index, it will return -1.
+function getAQI(data, index) {
+    let aqi = -1;
+    const row = data.data[index.row];
+    if (row) {
+        if (row[index.col]) {
+            aqi = row[index.col];
+        }
+    }
+    return aqi;
+}
+
+// Functionality for Clean Air Near Me feature
+
```

Comment:
extra line

Reply:
I left that there on purpose, because that comment isn't for the function but more of a section for me to group the clean air stuff under

Reply:
Can use /* ... */ to distinguish.